ouroboros-consensus-cardano/src/unstable-shelley-testlib/Test/Consensus/Shelley/Examples.hs

```diff
@@ -19,9 +19,12 @@ module Test.Consensus.Shelley.Examples (
   , examplesShelley
   ) where
 
+import           Data.List.NonEmpty
```

Comment:
Minor: Unqualified import of `Data.List.NonEmpty` causes a lot of conflicts with the prelude (
```suggestion
import qualified Data.List.NonEmpty as NE
```